Fix some grammar issues.

rowild opened the pull request:
Additionally to grammar fixes:
- Capitalize titles.
- Remove irrelevant text.

Questions:
"This procedure" => better: "This command"?
"[...] to use the Introduction with TYPO3 CMS 9.5 [...]" => "[...] to use the Introduction **Package** with TYPO3 CMS 9.5 [...]"?

This sentence does not make any sense:
"[...]`composer-typo3-auto-install` additionally takes care of the initial setup if
it's not already done, means no LocalConfiguration.php can be found. [...]"

I guess you mean:
"[...]`composer-typo3-auto-install` additionally takes care of the initial setup in case 
no `LocalConfiguration.php` can be found. [...]"

rowild committed:
Fix some grammar issues.

Additionally to grammar fixes:
- Capitalize titles.
- Remove irrelevant text.

Questions:
"This procedure" => better: "This command"?
"[...] to use the Introduction with TYPO3 CMS 9.5 [...]" => "[...] to use the Introduction **Package** with TYPO3 CMS 9.5 [...]"?

gilbertsoft commented on README.md:
```diff
@@ -2,7 +2,7 @@
 Get going quickly with TYPO3 CMS Introduction Package and DDEV Local.
 
-This repository provides a fully setup and ready to use TYPO3 CMS instance
+This repository provides a full setup and ready to use TYPO3 CMS instance
```
fully is correct here imho but set up should be written with two words instead.

rowild replied:
It is correct if "set up" is a verb, true. Do you want me to adapt that change?

rowild replied:
Will do :-)

gilbertsoft reviewed (changes requested):
Thanks for your suggestions, very appreciated!

rowild committed:
Incorporate Simon's feedback.

rowild committed:
Fix "setup" vs "set up" and its preceding "fully"

rowild committed:
Mini recommendations

Probably to be removed later on in case ddev provides a solution for automatic port handling.
